models/reference.py

```diff
@@ -74,24 +77,8 @@ class Reference(Base, AutoIDMixin):
         doc="Section ID of the reference image. "
     )
 
-    # this allows choosing a different reference for images taken before/after the validity time range
```

Comment:
The validity ranges would refer to the date of the new image, matching it to a reference based on the time range. RefSets will override that. In fact, it is unwanted behavior in which the data affects which reference gets picked. 

I guess we _could_ have a RefSet with validity ranges (that don't overlap) and then you'd get a different reference on year 1 and on year 2 from the same refset (like you can get references for different filters). But I think it would be much easier to just make a new refset.